src/presentational/image.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Image = props => (
+  <div>
+    Image
+    {/*Helpful button*/}
+    {/*Not Helpful button*/}
```

Comment:
Please don't add a button that doesn't do anything. Laurie Voss would not like that.

Reply:
Pseudo for "Helpful" and "Not Helpful" review feedback buttons that will be implemented post-proxy